orcaexample/kinetics.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import os
+
+from slowfast.models import build_model
+import slowfast.models.losses as losses
+import slowfast.models.optimizer as optim
+from slowfast.datasets import loader
+from slowfast.utils.parser import load_config
+
+from bigdl.dllib.utils.log4Error import invalidInputError
+from bigdl.orca.learn.pytorch import Estimator
+from bigdl.orca.learn.metrics import Accuracy
+from bigdl.orca.learn.trigger import EveryEpoch
+from bigdl.orca import init_orca_context, stop_orca_context
+
+
+parser = argparse.ArgumentParser(description='PyTorch Kinectics Example')
+parser.add_argument('--cluster_mode', type=str, default="local",
+                    help='The cluster mode, such as local, yarn-client, yarn-cluster, spark-submit or k8s.')
+parser.add_argument('--cfg_files', type=str, default="./SLOWFAST_8x8_R50.yaml",
+                    help='The path to config file')
+parser.add_argument('--backend', type=str, default="bigdl",
+                    help='The backend of PyTorch Estimator; bigdl, ray, and spark are supported')
+parser.add_argument("--executor_memory", type=str, default="8g", help="executor memory")
+parser.add_argument("--driver_memory", type=str, default="8g", help="driver memory")
+parser.add_argument(
+    "--opts",
+    help="See slowfast/config/defaults.py for all options",
+    default=None,
+    nargs=argparse.REMAINDER,
+)
+args = parser.parse_args()
+
+if args.cluster_mode == "local":
+    init_orca_context(memory="12g", cores=8)
+elif args.cluster_mode.startswith("yarn"):
+    hadoop_conf = os.environ.get("HADOOP_CONF_DIR")
+    invalidInputError(
```

Comment:
this is an internal API, not for external use. Just use `require` for user application code.

Reply:
Then the check of invalidInputError should be removed from example style check? Not sure if it is easy to do this.